Sources/Shared/Operators/Operators.swift

```diff
@@ -32,3 +33,7 @@ public func <+ <T>(inout left: [T], right: T?) {
   guard let right = right else { return }
   left.append(right)
 }
+
+public prefix func <? <T: DefaultType>(value: T?) -> T {
+  return value ?? T.defaultValue
+}
```

Comment:
Could you move the prefix operator up in the code so that all prefix operator declarations are grouped?